src/environments/assets.ts

```diff
@@ -1,28 +1,29 @@
 export const assets = [
-  'assets/release-notes-1.16.15.json',
   'assets/release-notes-1.19.0.json',
   'assets/release-notes-1.18.8.json',
   'assets/release-notes-1.18.7.json',
   'assets/release-notes-1.18.6.json',
+  'assets/release-notes-1.18.5.json',
+  'assets/release-notes-1.18.4.json',
+  'assets/release-notes-1.18.3.json',  
+  'assets/release-notes-1.18.2.json',
+  'assets/release-notes-1.18.1.json',
+  'assets/release-notes-1.18.0.json',
   'assets/release-notes-1.17.11.json',
   'assets/release-notes-1.17.10.json',
   'assets/release-notes-1.17.9.json',
-  'assets/release-notes-1.16.13.json',
   'assets/release-notes-1.17.8.json',
   'assets/release-notes-1.17.7.json',
   'assets/release-notes-1.17.6.json',
   'assets/release-notes-1.17.5.json',
-  'assets/release-notes-1.18.5.json',
-  'assets/release-notes-1.18.4.json',
-  'assets/release-notes-1.17.4.json',
-  'assets/release-notes-1.18.3.json',
+  'assets/release-notes-1.17.4.json',  
   'assets/release-notes-1.17.3.json',
-  'assets/release-notes-1.18.2.json',
-  'assets/release-notes-1.18.1.json',
-  'assets/release-notes-1.18.0.json',
   'assets/release-notes-1.17.2.json',
   'assets/release-notes-1.17.1.json',
   'assets/release-notes-1.17.0.json',
+  'assets/release-notes-1.16.15.json',
+  'assets/release-notes-1.16.14.json',
+  'assets/release-notes-1.16.13.json',
   'assets/release-notes-1.16.6.json',
   'assets/release-notes-1.16.5.json',
   'assets/release-notes-1.16.4.json',
```

Comment:
There are a lot of patch releases missing back from 1.16, should I generate those?

Reply:
Oh yeah that would be wonderful to have them complete here. :heart: